src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -447,7 +447,34 @@ namespace wallet_rpc
       };
       typedef epee::misc_utils::struct_init<response_t> response;
     };
+  
+  struct COMMAND_RPC_GET_INFO 
+  {
+    struct request_t
+    {
+      BEGIN_KV_SERIALIZE_MAP()
+      END_KV_SERIALIZE_MAP()
+    };
+    typedef epee::misc_utils::struct_init<request_t> request;
 
+      struct response_t
+      {
+        std::string  daemon_address;
+        std::string nettype;
+        bool mainnet;
```

Comment:
It rarely makes sense to include outputs of pure functions of response data in a response; exceptions can be made in case of complicated functions, but these boolean values are all trivially derived from the nettype (as can be seen by the code which sets them).

Reply:
I think `nettype` itself is enough. 

@iamamyth is that what you meant too? My apologies for asking, just wanted to clarify.

Reply:
> I think nettype itself is enough

Yes, that's what I meant.